pkg/acsengine/defaults-controller-manager.go

```diff
@@ -18,8 +18,9 @@ func setControllerManagerConfig(cs *api.ContainerService) {
 		"--cluster-signing-key-file":         "/etc/kubernetes/certs/ca.key",
 		"--service-account-private-key-file": "/etc/kubernetes/certs/apiserver.key",
 		"--leader-elect":                     "true",
-		"--v":                                "2",
-		"--profiling":                        "False",
+		"--use-service-account-credentials":  "true",
```

Comment:
Let's move `--use-service-account-credentials` down to the default struct so that users can opt out of it.

Reply:
Sure, actually I will remove this one since we already have a logic to set it when RBAC is on.